Inter-views/Inter-views
# Webpack and Dependencies initialization

## **Types of Changes**


## **Purpose**
initialize webpack and dependencies

## **Approach**
- install dependencies
- configure webpack with dev server 

## **How to Test**
- npm run build // should create dist folder with bundled files
- npm start // should start app in production mode on port 3000
- npm run dev // should open tab on 8000, proxied to 3000, with HMR
